### PR TITLE
Update with v15.0.2 release targets

### PR DESCRIPTION
**Pull request by vpnguye2**
This PR updates the template to refer to v15.0.2 release artifacts

### CODE REVIEW

File: cumulus-tf/terraform.tfvars.example [hunk 1]
```diff
@@ -115,11 +115,17 @@ tea_distribution_url = "TEA distribution url"
 
 # Cumulus Distribution variables.
 
-deploy_cumulus_distribution = false
+deploy_cumulus_distribution = true
```
Comment (jennyhliu) [inline cumulus-tf/terraform.tfvars.example:118]: why?  I think the default is still tea

File: cumulus-tf/main.tf [hunk 2]
```diff
@@ -108,6 +108,9 @@ module "cumulus" {
 
   dynamo_tables = data.terraform_remote_state.data_persistence.outputs.dynamo_tables
 
+  default_log_retention_days = var.default_log_retention_days
```
Comment (jennyhliu) [inline cumulus-tf/main.tf:111]: make them optional.
```
# optional
# default_log_retention_days = var.default_log_retention_days
# cloudwatch_log_retention_periods = var.cloudwatch_log_retention_periods
```

File: cumulus-tf/thin_egress_app.tf [hunk 3]
```diff
@@ -62,7 +62,7 @@ resource "aws_cloudwatch_log_subscription_filter" "egress_api_gateway_log_subscr
 resource "aws_cloudwatch_log_group" "egress_lambda_log_group" {
   count             = (var.log_destination_arn != null) ? 1 : 0
   name              = "/aws/lambda/${module.thin_egress_app.egress_lambda_name}"
-  retention_in_days = 30
+  retention_in_days = lookup(var.cloudwatch_log_retention_periods, "thin-egress-app-EgressLambda", var.default_log_retention_days)
```
Comment (jennyhliu) [inline cumulus-tf/thin_egress_app.tf:65]: reverse the change, it's up to the DAAC to configure this.

File: cumulus-tf/variables.tf [hunk 4]
```diff
@@ -268,3 +268,34 @@ variable "deploy_cumulus_distribution" {
   type        = bool
   default     = false
 }
+
+variable "cloudwatch_log_retention_periods" {
+  type = map(number)
+  description = "retention periods for the respective cloudwatch log group, these values will be used instead of default retention days"
+  default = {
```
Comment (jennyhliu) [inline cumulus-tf/variables.tf:275]: these values are for cumulus example, change these two variables as in
https://github.com/nasa/cumulus/blob/v15.0.2/tf-modules/cumulus/variables.tf#L602-L612

File: cumulus-tf/cumulus_distribution.tf.example [hunk 5]
```diff
@@ -38,4 +38,6 @@ module "cumulus_distribution" {
   system_bucket            = var.system_bucket
   tags                     = local.tags
   vpc_id                   = var.vpc_id
```
Comment (jennyhliu) [inline cumulus-tf/cumulus_distribution.tf.example:40]: make them optional

File: cumulus-tf/thin_egress_app.tf (hunk 3 above)
Comment (vpnguye2) [reply cumulus-tf/thin_egress_app.tf:65]: reversed the change.

File: cumulus-tf/cumulus_distribution.tf.example (hunk 5 above)
Comment (vpnguye2) [reply cumulus-tf/cumulus_distribution.tf.example:40]: Updated

File: cumulus-tf/main.tf (hunk 2 above)
Comment (vpnguye2) [reply cumulus-tf/main.tf:111]: Updated

File: cumulus-tf/terraform.tfvars.example (hunk 1 above)
Comment (vpnguye2) [reply cumulus-tf/terraform.tfvars.example:118]: Updated

File: cumulus-tf/variables.tf (hunk 4 above)
Comment (vpnguye2) [reply cumulus-tf/variables.tf:275]: Updated as commented

**Review by jennyhliu — APPROVED** [review]

**Pull request merged**